src/service/transaction-service.ts

```diff
@@ -121,6 +122,7 @@ export function parseGetTransactionsFilters(req: RequestWithToken): TransactionF
     fromDate: asDate(req.query.fromDate),
     tillDate: asDate(req.query.tillDate),
     invoiceId: asNumber(req.query.invoiceId),
+    excludeOwnTransaction: asNumber(req.query.excludeOwnTransaction),
```

Comment:
het heet `excludeOwn` maar het is een feiten een id die je toch exclude? dus dan is het toch meer een `excludeById`?